### ISSUE TITLE
arcface-torch model zoom only have cosface pre-trained model of glint360，I can use cosface pre-trained model  to train arcface 。

### ISSUE THREAD

**Issue by create-li** [deepinsight/insightface#1665]
I try to use cosface pre-trained model of glint360 to train the arcface，why the loss is very large, the loss value is 48.0 at the beginning, shouldn’t it be a relatively small loss?

**Comment by Henry-Schuetze** [comment]
The initially high loss in your scenario might be due the fact that (even though you have a pretrained backbone from cosface training) you probably "installed" a new (arcface) softmax layer in your model which still needs to be trained from scratch.

Despite an initially high loss, the convergence of your model should be much faster as compared to train the whole model from scratch.
